### PR TITLE
Fix CS by removing inline assignment.

### PR DESCRIPTION
**Pull request by dereuromark**
Follow https://github.com/propelorm/Propel2/pull/1944

The remaining issues need some code refactoring.

### CODE REVIEW

**Review by gechetspr — APPROVED** [review]

File: src/Propel/Generator/Builder/Om/TableMapBuilder.php [hunk 1]
```diff
@@ -34,18 +34,17 @@ public function getPackage(): string
...
     public function getNamespace(): ?string
     {
-        if (!$namespace = parent::getNamespace()) {
+        $namespace = parent::getNamespace();
+        if (!$namespace) {
             return 'Map';
         }
 
-        if (
-            $this->getGeneratorConfig()
```
Comment (limeeugenia) [inline src/Propel/Generator/Builder/Om/TableMapBuilder.php:42]: This `$this->getGeneratorConfig()` is lost in this edit.

File: src/Propel/Generator/Platform/SqlitePlatform.php [hunk 2]
```diff
@@ -517,7 +519,8 @@ public function getAddTableDDL(Table $table): string
             $lines[] = $this->getColumnDDL($column);
         }
 
-        if ($table->hasPrimaryKey() && ($pk = $this->getPrimaryKeyDDL($table))) {
+        $pk = $this->getPrimaryKeyDDL($table);
```
Comment (limeeugenia) [inline src/Propel/Generator/Platform/SqlitePlatform.php:522]: This change seems a logic change.

File: src/Propel/Generator/Builder/Om/TableMapBuilder.php (hunk 1 above)
Comment (dereuromark) [reply src/Propel/Generator/Builder/Om/TableMapBuilder.php:42]: On purpose
The internal method does that lookup :)

File: src/Propel/Generator/Platform/SqlitePlatform.php (hunk 2 above)
Comment (dereuromark) [reply src/Propel/Generator/Platform/SqlitePlatform.php:522]: Nope, see internal method.

**Review by limeeugenia — APPROVED** [review]

**Pull request merged**